line-sdk/build.gradle

```diff
@@ -7,7 +7,7 @@ apply plugin: 'maven-publish'
 apply plugin: 'signing'
 
 group = "com.linecorp.linesdk"
-version = "5.6.2"
+version = "5.6.3"
```

Comment:
Since it is adding new interface to the current project. Let's make it a minor update instead of a patch one!